web/src/app/ui/kmwuibutton.ts

```diff
@@ -293,7 +293,7 @@ if(!keyman?.ui?.name) {
 
         /* TODO: why is this still needed??? Does it actually do anything?? */
         osk.addEventListener('hide', function(obj) {
-          if((arguments.length > 0) && obj['hiddenByUser']) {
+          if((arguments.length > 0) && obj['HiddenByUser']) {
```

Comment:
Is this a bug fix?

Reply:
Brought to you by `noImplicitAny`.  It could detect typings after the fix, but not before.